src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -33,6 +33,8 @@ warning.asoc.certificates = Allowing Untrusted Connections is not applied to thi
 error.application.empty.ui = Application list is empty. Please check the credentials for service connection. 
 error.url.validation.ui = Unless this is a private network, please verify your domain.
 error.url.validation = An error occurred while validating the Starting URL: {0}.
+error.active.subscription.validation = You do not have a valid {0} subscription.
+error.active.subscription.validation.ui = You do not have subscription for selected scan type.
```

Comment:
Can you update this string to be:
"You do not have **a** subscription for **the** selected scan type."

Reply:
Okay, sure Matt.